source/Grid/Grid.js

```diff
@@ -381,6 +384,31 @@ export default class Grid extends PureComponent {
     })
   }
 
+  /**
+   * Ensure offset position is visible
+   * Useful for animating position changes
+   */
+  scrollToPosition ({
+    scrollLeft = 0,
+    scrollTop = 0
```

Comment:
I don't think `scrollLeft` or `scrollTop` should have default values of 0. That will make them _required_ parameters. The `_setScrollPosition` helper method allows either param to be omitted, in which case the current value in `state` won't be modified. This seems convenient if you want to scroll one axis of `Grid` without affecting the other.